ui/pages/home/home.component.js

```diff
@@ -504,7 +506,10 @@ export default class Home extends PureComponent {
       return null;
     }
 
-    const showWhatsNew = notificationsToShow && showWhatsNewPopup;
+    const showWhatsNew =
```

Comment:
We still want to show the what's new popup to users who already have metamask installed, upon version updates.

What happens is that if there are new "What's New" messages that we want to show to all users, then we update the notifications json file in app. After opening MetaMask the first time after the version updates, all users will have `notificationsToShow` and will see the popup.

So we need to somehow change `firstTimeFlowType === 'import'` to`(userHasJustOnboarded && firstTimeFlowType === 'import') || (!userHasJustOnboarded)`

Off the top of my head I am not sure if we have anything already in state to indicate whether `userHasJustOnboarded`... let me know if you can't find anything, it should be fairly straightforward to add

Reply:
We have this in metamask state `completedOnboarding`, maybe we can use that for this issue? What do you think about this @danjm?

Reply:
It looks like we can use `completedOnboarding` as it is set here in the `store` https://github.com/MetaMask/metamask-extension/blob/4a6585fc4a1f4c10ec376062a87b8d0ee5f6633c/app/scripts/controllers/onboarding.js#L49-L54

Reply:
@danjm @david0xd I used `completedOnboarding` for update `firstTimeFlowType === 'import'` to `(completedOnboarding && firstTimeFlowType === 'import') || (!completedOnboarding)`.
[Here](f2d603773d9638c337e34b039463edfd7644e3d2) I am changed this case.